Formula/etcher-cli.rb

```diff
@@ -0,0 +1,28 @@
+require "language/node"
+class EtcherCli < Formula
+
+  desc "Flash OS images to SD cards & USB drives, safely and easily"
+  homepage "https://etcher.io/"
+  url "https://github.com/resin-io/etcher/archive/v1.4.4.tar.gz",
```

Comment:
Formulae should have either a `url` and `sha256` or a `url`, `tag` and `revision`. Don't need both kinds of URL.